### ISSUE TITLE
Included pdf misses elements of text/figure text

### ISSUE THREAD

**Issue by fzeiser** [uio-latex/phduio-article-based#3]
I included my articles with the `includearticle` command. Unfortunately, the (axis) text of some figures was messed up after including the pdf. The problem was very similar to the description here on stackexchage (just that only some letters were missing):

 https://tex.stackexchange.com/questions/129309/pdf-figure-missing-text-elements-in-beamer-presentation

For me, adding `\pdfinclusioncopyfonts=1` in the preable (or probably also in the style file) fixed the problem. I thought that this might be interesting for others, too and might even be added to the FAQ. [Unfortunately, I didn't check the figures thoroughly enough and ended up printing this version.]

I attach screenshots of a figure in the article that turned out weird.

Wrong image (when included via `includearticle`):
![image](https://user-images.githubusercontent.com/15156776/110024030-28b74880-7d2e-11eb-9441-2dba43a445ef.png)

Correct / original image / article
![image](https://user-images.githubusercontent.com/15156776/110024103-3d93dc00-7d2e-11eb-88d9-2aa56ced5794.png)

**Comment by martinhelso** [comment]
This is not really specific to the `phduio` template, but I will add a comment in the FAQ.

Out of curiosity, did you work on a UiO Linux machine? The older template `uiophd` claims that [font embedding is automatically taken care](https://www.mn.uio.no/ifi/tjenester/it/hjelp/latex/uiophd.html) of on those computers.

**Comment by fzeiser** [comment]
I totally agree, I just thought that it is helpful anyhow. 👍 

> Out of curiosity, did you work on a UiO Linux machine? The older template uiophd claims that font embedding is automatically taken care of on those computers.

No, I did not. I worked on a Linux mint ( Ubuntu like) machine. 

It’s nothing I had experienced before, even tough I typeset also the original article I included in the theses (where the problem did not occur). So only once I included the article with the pdf figure as a pdf in the thesis the problem occurred. I don’t know enough about what `includegraphics ` calls internally, but the behavior seems to be different to the commands underlying `includearticle`.
